tests/features/collection_page/api.feature

```diff
@@ -0,0 +1,23 @@
+@api
+Feature: Collection Page API
+  In order to manage collection pages programmatically
+  As a backend developer
+  I need to be able to use the Node API to handle the "Collection Page" bundle
```

Comment:
Node API? :) What's that? :)

Just change it to something like "I need to be able to create, edit and delete collection pages through the API"